e2e/src/editCra.e2e-spec.ts

```diff
@@ -151,6 +154,34 @@ describe('When I input my timesheet', () => {
       it('should give me two links', () => {
         expect(editCraPage.getElementsByText('ce lien').count()).toBe(2);
       });
+
+      describe('When I click on one of these links', () => {
+
+        it('should get data from the link', () => {
+          editCraPage.getElementsByText('ce lien').get(0).getAttribute('href').then((value: string) => {
+            browser.get(value);
+            expect(nameInput.getAttribute('value')).toBe('Tester');
+            expect(emailInput.getAttribute('value')).toBe('tester@test.com');
+            expect(titleInput.getAttribute('value')).toBe('Testing');
+            expect(clientInput.getAttribute('value')).toBe('Test.com');
+            expect(selectedNumber.getText()).toBe(getBusinessDaysNumber().toString());
+          });
+        });
+
+        it('should be the edit link if it was the first one', () => {
```

Comment:
this does not make any sense

Reply:
I agree, this `it` has no sense, use one `describe` for each link instead of `When I click on one of these links` to be clearer